### PR TITLE
migrate to AutoPlugin

### PR DESCRIPTION
**Pull request by xuwei-k**
fix https://github.com/sbt/sbt-appengine/issues/39

[diff without indent changes](https://github.com/sbt/sbt-appengine/pull/45/files?w=1)

### CODE REVIEW

File: src/main/scala/AppenginePlugin.scala [hunk 1]
```diff
@@ -2,64 +2,82 @@ package sbtappengine
...
-    lazy val localDbPath    = SettingKey[File]("appengine-local-db-path")
-    lazy val debug          = SettingKey[Boolean]("appengine-debug")
-    lazy val debugPort      = SettingKey[Int]("appengine-debug-port")
-    lazy val includeLibUser = SettingKey[Boolean]("appengine-include-lib-user")
-    lazy val persistenceApi = SettingKey[String]("appengine-persistence-api", "Name of the API we are enhancing for: JDO, JPA.")
+  override def requires = sbt.plugins.JvmPlugin
+
+  object autoImport {
+    object AppengineKeys extends revolver.RevolverKeys {
+      lazy val requestLogs    = InputKey[Unit]("appengine-request-logs", "Write request logs in Apache common log format.")
```
Comment (eed3si9n) [inline src/main/scala/AppenginePlugin.scala]: Key names should be renamed to `appengineRequestLogs` so they'll look the same on both build.sbt and the shell.

Comment (xuwei-k) [inline src/main/scala/AppenginePlugin.scala]: OK, will do. also we should remove AppengineKeys object ?

Comment (xuwei-k) [inline src/main/scala/AppenginePlugin.scala]: done https://github.com/sbt/sbt-appengine/pull/45/commits/7ba902f843fbafe19f386db4cc8f09af61a736a7

**Comment by xuwei-k** [comment]
I will release 0.7.0

**Comment by eed3si9n** [comment]
Sounds good. We can both use https://bintray.com/sbt/sbt-plugin-releases/sbt-appengine

**Pull request merged**